provisionerd/runner/runner.go

```diff
@@ -586,19 +587,19 @@ func (r *Runner) runTemplateImport(ctx context.Context) (*proto.CompletedJob, *p
 
 // Parses template variables and README from source.
 func (r *Runner) runTemplateImportParse(ctx context.Context) (
-	vars []*sdkproto.TemplateVariable, readme []byte, err error,
+	workspaceTags map[string]string, vars []*sdkproto.TemplateVariable, readme []byte, err error,
```

Comment:
Are we reaching the stage where should return a `struct, error` instead?

Reply:
I might be wrong, but wasn't the golden limit = 4? I guess this is the last time :)